functions.php

```diff
@@ -47,6 +59,9 @@ function prepareOutput(array $query): array
 {
     $result = [];
     foreach ($query as $el) {
+        if ($el['deleted']) {
```

Comment:
Although this does work, it would be better to let SQL filter the results for you by adding a where condition that makes sure deleted is 0